MHCollapsibleViewManager.m

```diff
@@ -212,7 +298,7 @@ -(MHTableViewCell*)returnCellWithIndex:(NSIndexPath*)indexPath tableView:(UITabl
         type = CRUCellViewInteractionHeader;
         collapsed = self.expanded;
         cellText = self.headerTitle;
-        detailText = [NSString stringWithFormat:NSLocalizedString(@"%i items",), self.filterSections.count];
```

Comment:
That's good to know, thanks! I will work on the string setup to be as you ask. Is there a way to check for english? I added a piece that handles plural versus singleton since it could be 1 item instead of items:
```
- (NSString*)detailedHeaderSectionText{
    
    NSUInteger count = [self selectedCountForSubTitleText];
    NSString *detailedText;
    NSString *itemTitle = self.identifier;
    NSString *plural = @"";
    NSString *selected = @"selected";
    
    if(itemTitle == nil){
        itemTitle = @"item";
    }
    
    if(count < 1){
        selected = @"";
        itemTitle = @"item";
        count = self.itemCount;
        if(count > 1){
            plural = @"s";
        }
    }
    else{
        if(count > 1){
            plural = @"s";
        }
    }
    detailedText = [NSString stringWithFormat:NSLocalizedString(@"%i %@ %@%@",), count, selected, itemTitle, plural];
    itemTitle = nil;
    plural = nil;
    selected = nil;
    return detailedText;
}
```